subnet/add_validator_subnet.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package subnet
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/avalanche"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/multisig"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
+	"golang.org/x/net/context"
+)
+
+type ValidatorParams struct {
+	NodeID ids.NodeID
+
+	Duration time.Duration
+
+	Weight uint64
+
+	Network avalanche.Network
+}
+
+// AddValidator adds validator to subnet
+// Before an Avalanche Node can be added as a validator to a Subnet, the node must already be
+// tracking the subnet
+// TODO: add more description once node join subnet sdk is done
+func (c *Subnet) AddValidator(wallet wallet.Wallet, validatorInput ValidatorParams) (*multisig.Multisig, error) {
+	if validatorInput.NodeID == ids.EmptyNodeID {
+		return nil, fmt.Errorf("validator node id is not provided")
```

Comment:
please provide public err constants

Reply:
not sure wether to use fmt.Errorf or errors.New for new errors without formatting involved. probably errors.New is more estandard

Reply:
could you provide error constants? if possible, public. A library is expected to return errors that can be compared with error constants provided by the library